### PR TITLE
Document minor enhancements

### PR DESCRIPTION
**Pull request by Tsvetomir-Hr**
Related to: https://github.com/telerik/blazor/issues/9737

### CODE REVIEW

File: components/card/overview.md [hunk 1]
```diff
@@ -62,11 +62,12 @@ The Card provides various parameters that allow you to configure the component:
 
 | Parameter   | Type | Description |
 | ----------- | ----------- | -------|
-| `Width` | `string` | defines width of the component.
 | `Class` | `string` | the CSS class that will be rendered on the main wrapping element of the Card.
```
Comment (yordan-mitev) [inline components/card/overview.md:65]: The content of the `Description` column requires a minor update. The first letter must be capitalized on all rows. See the `components/carousel/overview.md` file in this PR, for example.

File: components/card/overview.md [hunk 2]
```diff
@@ -62,11 +62,12 @@ The Card provides various parameters that allow you to configure the component:
...
+| `ChildContent` | `RenderFragment` | defines the child content of the component. All possible building blocks can be directly used as a `ChildContent` of the Card.
```
Comment (yordan-mitev) [inline components/card/overview.md:66]: Make "building blocks" an anchor that leads to the Building Blocks article.

File: components/pager/overview.md [hunk 3]
```diff
@@ -87,6 +87,7 @@ The Blazor Pager provides various parameters that allow you to configure the com
 | `PageSize` | `int` | The number of items to display on a page. Supports two-way binding. |
 | `PageSizes` | `List<int?>` | Allows users to change the page size via a DropDownList. The attribute configures the DropDownList options. A `null` item in the `PageSizes` `List` will render an "All" option. By default, the Pager DropDownList is not displayed. You can also set `PageSizes` to `null` programmatically to remove the DropDownList at any time. |
 | `InputType` | `PagerInputType` enum <br /> (`Buttons`) | Determines if the pager will show numeric buttons to go to a specific page, or a textbox to type the page index. The arrow buttons are always visible. The `PagerInputType` enum accepts values `Buttons` (default) or `Input`. When `Input` is used, the page index will change when the textbox is blurred, or when the user hits Enter. This is to avoid unintentional data requests. |
+| `ShowInfo` | `bool` |Defines whether the information about the current page and the total number of records is present. Default value is `true`.|
```
Comment (svdimitr) [inline components/pager/overview.md:90]: We include the default value in the `Type` column.

File: components/card/overview.md [hunk 4]
```diff
@@ -62,11 +62,12 @@ The Card provides various parameters that allow you to configure the component:
...
-| `Orientation` | `CardOrientation` | defines the orientation of the card. Takes a member of the `Telerik.Blazor.CardOrientation` enum (`Horizontal` or `Vertical`). Read more in the [Card Orientation article]({%slug card-orientation%}).
-| `ThemeColor` | `string` | defines the appearance of the component. We support predefined theme colors such as primary, secondary, tertiary, success, info, warning, error, dark, light, and inverse (members of the `Telerik.Blazor.ThemeConstants.Card` class). Test changing the Card theme colors in our [live demo](https://demos.telerik.com/blazor-ui/card/appearance).
-| `ChildContent` | `RenderFragment` | defines the child content of the component. All possible building blocks can be directly used as a `ChildContent` of the Card.
+| `Class` | `string` | The CSS class that will be rendered on the main wrapping element of the Card.
+| `ChildContent` | `RenderFragment` | Defines the child content of the component. All possible [Card Building Blocks]({%slug card-building-blocks%}) can be directly used as a `ChildContent` of the Card.
+| `Height` | `string` | Defines height of the component.
```
Comment (svdimitr) [inline components/card/overview.md:67]: Defines **the** height of the component seems more fluent to me.

File: components/carousel/overview.md [hunk 5]
```diff
@@ -82,6 +82,7 @@ The following table lists Carousel parameters. Check the [Carousel API Reference
 | `Width` | `string` | The Carousel width. See [Dimensions]({%slug common-features/dimensions%}) for more details. The Carousel renders in a `<div>`, so it expands horizontally to 100% by default.|
 | `Height` | `string` | The Carousel height. By default and by design, the component has no height and does not expand, based on its content. *In other words, the Carousel will be zero pixels high, if height is not applied.*|
 | `Class` | `string` | The CSS class that will be rendered on the main wrapping element of the component. Use it to apply custom styles or [override the theme]({%slug themes-override%}).
+| `ThemeColor` | `string` |Defines the theme color of the Carousel. Default value is `ThemeConstants.Carousel.ThemeColor.Light`.|
```
Comment (svdimitr) [inline components/carousel/overview.md:85]: The same feedback as for the `ShowInfo`

**Review by yordan-mitev — APPROVED** [review]

File: components/carousel/overview.md [hunk 6]
```diff
@@ -82,6 +82,7 @@ The following table lists Carousel parameters. Check the [Carousel API Reference
 | `Width` | `string` | The Carousel width. See [Dimensions]({%slug common-features/dimensions%}) for more details. The Carousel renders in a `<div>`, so it expands horizontally to 100% by default.|
 | `Height` | `string` | The Carousel height. By default and by design, the component has no height and does not expand, based on its content. *In other words, the Carousel will be zero pixels high, if height is not applied.*|
 | `Class` | `string` | The CSS class that will be rendered on the main wrapping element of the component. Use it to apply custom styles or [override the theme]({%slug themes-override%}).
+| `ThemeColor` | `ThemeConstants.Carousel.ThemeColor` string <br/> (`Light`) |Defines the theme color of the Carousel component.
```
Comment (dimodi) [inline components/carousel/overview.md:85]: * The current implementation makes the type look like `enum`, but it isn't. Technically, the default value of the property is `"light"` with lowercase "l".
* Use `|` at the end of table rows when other rows have it.
* Try not to explain property names with the same words.

```suggestion
| `ThemeColor` | string <br /> (`"light"`) | Sets predefined colors to the Carousel component. Use the members of the static class `ThemeConstants.Carousel.ThemeColor` to set valid values. |
```

**Pull request merged**